Ednpoints.txt

```diff
@@ -0,0 +1,132 @@
+The same routes for users, chefs and delivery
+POST: /auth/register - registration
+POST: /auth/login - enter to account
+GET: /auth/logout - exit from account
```

Comment:
common practice is to use POST method for logging out request
https://stackoverflow.com/questions/29827251/should-a-logoff-request-be-designed-as-get-or-post

Reply:
Okay, thanks, it's clear